### ISSUE TITLE
Add FAQ and Troubleshooting pages

### ISSUE THREAD

**Issue by jni** [napari/docs#493]
## 📚 New content request

We should have two pages: one of miscellaneous questions, and another of common errors and how to work around them. Examples:

- troubleshooting: torch + rosetta + napari crashes Python: napari/napari#7259
- troubleshooting: "module napari has no attribute Viewer" when doing an editable install on top of a normal one, or when you have a local file `napari.py`
- faq: how do I use napari in a remote Jupyter env?

**Comment by psobolewskiPhD** [comment]
> how do I use napari in a remote Jupyter env?

This one (on my to-do) is probably more than a FAQ answer I think...

**Comment by jni** [comment]
Yes, sorry, in this case I meant it would have a quick response "napari is a native app so it's not possible to do this directly within a notebook or lab tab, we are aware of this issue, but currently there are some options to use napari in a remote desktop, see this page" [include a link to a page with all the options].

**Comment by jni** [comment]
Another one:
- my image renders as all black / all white (inspired by napari/napari#5136: it would be good if user queries in those situations can be pointed to exactly one link discussing troubleshooting for this. Like the remote jupyter question, this could have a quick answer, and a link to a longer guide about contrast limits.)

**Comment by jni** [comment]
Another: napari + numba combination doesn't work on admin installs of Windows (napari/napari#7288), but the fix is relatively easy (set NUMBA_CACHE_DIR).

**Comment by psobolewskiPhD** [comment]
We have a troubleshooting page:
https://napari.org/stable/troubleshooting.html

**Comment by melissawm** [comment]
Adding the FAQ to the agenda for the next docs working group meeting.
